bin/resources/GameIndex.yaml

```diff
@@ -9193,7 +9214,7 @@ SLES-50193:
   region: "PAL-E"
   compat: 5
 SLES-50194:
-  name: "4x4 Evolution"
+  name: "4x4 Evo"
```

Comment:
subjective, if you look at the front of the cover it says 4x4 EVO, but you look on the spine or back, it says Evolution http://www.thecoverproject.net/view.php?cover_id=5743

Reply:
Take a look at the title screen:
https://www.youtube.com/watch?v=Vx6yF8240Y8

Reply:
As I said, it depends where you look :P That's the logo, but the full title of the game on the side of the box is 4x4 Evolution